src/scripts/cards.js

```diff
@@ -0,0 +1,15 @@
+import { getArticles } from './requests.js';
+import { changeViewClassForAllCards, createArticleHTML } from './utils.js';
+
+const articles__content = document.querySelector('.articles__content');
+
+getArticles().then(renderArticles);
+
+function renderArticles(data) {
+  articles__content.innerHTML = data
+    .slice(0, 6)
+    .map((article) => createArticleHTML(article))
+    .join('');
```

Comment:
это если сделать так, чтобы функция createArticleHTML возвращала node узел
просто идея когда вы перезатираете innerHtml не очень

ну или используйте insertAdjacentHTML

```suggestion
const article_items = data.slice(0, 6).map((article) => createArticleHTML(article)) // получили массив node
articles__content.append(article_items)
```